dasharo/project_config.yaml

```diff
@@ -1,40 +1,44 @@
 # Dasharo project configuration data
-file_version: 0.0.1
+file_version: 1.0.0
 
 dasharo:
-  platform: novacustom_ns5x
-  newsletter:
-    url:  https://newsletter.3mdeb.com/subscription/T61MyO2sP
+        platform: novacustom_nv4x
+        newsletter:
+                url:  https://newsletter.3mdeb.com/subscription/T61MyO2sP
 
 client:
-  name: NovaCustom
+        name: NovaCustom
 
 framework:
-  name: coreboot
-  version: 4.16
-  url_template: https://github.com/Dasharo/coreboot/tree/COMMIT_HASH
+        name: coreboot
+        version: ae9a8447
```

Comment:
This is the latest upstream commit which the code is based on?

Reply:
No, fixed.